Dockerfile

```diff
@@ -18,6 +20,8 @@ WORKDIR /application
 COPY /dependencies.sh /application/
 RUN /application/dependencies.sh
 
+COPY --from=BUILD /application/database /application/database
+
 COPY --from=BUILD /application/bin/${PROJECT_NAME} /application/bin/${PROJECT_NAME}
 COPY --from=BUILD /application/bin/${PROJECT_NAME}-tests /application/bin/${PROJECT_NAME}-tests
 
```

Comment:
Suggestion (needs some adjustment)

# Use an official Ubuntu Mantic release as the base image for building
FROM ubuntu:mantic as BUILD

# Set environment variables
ENV PROJECT_NAME=desktop-business-app

# Set the working directory inside the container
WORKDIR /application

# Copy dependency installation script and install dependencies
# COPY dependencies.sh /application/

# Copy the rest of the application code to the working directory
COPY . /application
RUN chmod +x /application/dependencies.sh && /application/dependencies.sh

# Build the project using CMake
RUN cmake -S . -B build && cmake --build build

# Run tests
RUN /application/bin/${PROJECT_NAME}-tests

# Use a separate stage for the final image
FROM ubuntu:mantic as FINAL

# Set environment variables
ENV PROJECT_NAME=desktop-business-app

# Set the working directory inside the container
WORKDIR /application
COPY . /application
# Copy dependency installation script and install dependencies
# COPY dependencies.sh /application/
RUN chmod +x /application/dependencies.sh && /application/dependencies.sh

# Copy only the necessary files from the build stage
COPY --from=BUILD /application/build/${PROJECT_NAME} /application/bin/${PROJECT_NAME}
COPY --from=BUILD /application/build/${PROJECT_NAME}-tests /application/bin/${PROJECT_NAME}-tests

# Set the entry point for the container
ENTRYPOINT ["/application/bin/desktop-business-app"]